source/privacy/index.markdown

```diff
@@ -46,13 +46,17 @@ If Home Assistant, or substantially all of its assets, were acquired, or in the
 
 Ads appearing on any of our websites may be delivered to users by advertising partners, who may set cookies. These cookies allow the ad server to recognize your computer each time they send you an online advertisement to compile information about you or others who use your computer. This information allows ad networks to, among other things, deliver targeted advertisements that they believe will be of most interest to you. This Privacy Policy covers the use of cookies by Home Assistant and does not cover the use of cookies by any advertisers.
 
+## Companion Applications
+All information collected by the applications are sent directly to your hosted instance of Home Assistant.  This includes, but is not limited to, sensor data and location data, which at times is sent while the application is in the background.  At no point is this data logged or given to a 3rd party.
```

Comment:
Missing space after the header, `are` -> `is`, and no double spaces.

Cloud you open a PR to correct that @JBassett?  Thanks 👍 

```
## Companion Applications

All information collected by the applications is sent directly to your hosted instance of Home Assistant. This includes, but is not limited to, sensor data and location data, which at times is sent while the application is in the background. At no point is this data logged or given to a 3rd party.
```